### PR TITLE
Add `CMyFont`

### PR DESCRIPTION
**Pull request by Renzo904**
Original files from:
[CMyFont.hpp](https://t-pot.com/program/35_quaternion/font_h.html)
[CMyFont.cpp](https://t-pot.com/program/35_quaternion/font_cpp.html)

### CODE REVIEW

File: scripts/configure.py [hunk 1]
```diff
@@ -104,6 +104,7 @@ def configure(build_type):
             "GameManager",
             "Chain",
             "Controller",
+            "CMyFont",
```
Comment (roblabla) [inline scripts/configure.py:107]: This should go between Chain and Controller so it ends up in the right location in the executable, I believe?

**Comment by wearrrrr** [comment]
Looks good to me :)

**Comment by KSSBrawl** [comment]
It might be prudent to replace `WIDTH` and `HEIGHT` with `GAME_WINDOW_WIDTH` and `GAME_WINDOW_HEIGHT` from `GameWindow.hpp` since in the original `font.cpp` those seem to be similar configurable values.

**Comment by Renzo904** [comment]
> It might be prudent to replace `WIDTH` and `HEIGHT` with `GAME_WINDOW_WIDTH` and `GAME_WINDOW_HEIGHT` from `GameWindow.hpp` since in the original `font.cpp` those seem to be similar configurable values.

To be honest, i totally forgot those existed😂

**Pull request merged**